branca/element.py

```diff
@@ -43,11 +43,11 @@ def __init__(self, template=None, template_name=None):
         self._parent = None
         self._template = Template(template) if template is not None\
             else ENV.get_template(template_name) if template_name is not None\
-            else Template(u"""
```

Comment:
I would avoid the line breaks you have above, but don't worry about that.  It is just style.